src/snapred/ui/handler/SNAPResponseHandler.py

```diff
@@ -33,11 +33,13 @@ def _handle(self, result):
             if userSelectedContinueAnyway:
                 self.continueAnyway.emit()
         else:
-            self.rethrow()
+            self.rethrow(result)
 
+    @staticmethod
     def _isErrorCode(code):
         return code >= ResponseCode.ERROR
 
+    @staticmethod
     def _isRecoverableError(code):
         return ResponseCode.RECOVERABLE <= code < ResponseCode.ERROR
```

Comment:
This kind of _logic_ statement depends on precedence rules for parsing.  Personally, I never remember precedence rules so I have _no_ _idea_ what this does.  Does it mean `(ResponseCode.RECOVERABLE <= code) and (code < ResponseCode.ERROR)`  OR does it mean (when `code < ResponseCode.ERROR` => `ResponseCode.RECOVERABLE < True`...  Do you see what I mean?

Reply:
This is essentially equivalent to: 

```python
(ResponseCode.RECOVERABLE <= code) and (code < ResponseCode.ERROR)
``` 

This means the statement checks if `code` is greater than or equal to `Response.RECOVERABLE` and also less than `ResponseCode.Error`.

It does **NOT** mean:
```python
when code < ResponseCode.ERROR => ResponseCode.RECOVERABLE < True
```

This is left this way in the implementation so that more Recoverable errors can be assigned to response codes that are between greater than 400 (recoverable) and less than 500 (error). State initialization is allocated to 400 or 401 I think. But other recoverable errors and subsequent divergent workflow branches can be allocated properly later on.

Reply:
I'm fine with this.  It's probably more Pythonic anyway; I have just never seen it before.   I need to take a look at the comparison operator syntax and see what it actually allows.  :)